tests/components/zha/test_registries.py

```diff
@@ -440,23 +441,13 @@ def test_quirk_classes() -> None:
 
     def find_quirk_class(base_obj, quirk_mod, quirk_cls):
         """Find a specific quirk class."""
-        mods = dict(inspect.getmembers(base_obj, inspect.ismodule))
-
-        # Check if we have found the right module
-        if quirk_mod in mods:
-            # If so, look for the class
-            clss = dict(inspect.getmembers(mods[quirk_mod], inspect.isclass))
-            if quirk_cls in clss:
-                # Quirk class found
-                return True
-
-        else:
-            # Recurse into other modules
-            for mod in mods:
-                if not mods[mod].__name__.startswith("zhaquirks."):
-                    continue
-                if find_quirk_class(mods[mod], quirk_mod, quirk_cls):
-                    return True
+
+        module = importlib.import_module(quirk_mod)
+        clss = dict(inspect.getmembers(module, inspect.isclass))
+        if quirk_cls in clss:
+            # Quirk class found
+            return True
+
         return False
```

Comment:
Can probably be shortened now:
```suggestion
        # Quirk class found
        return quirk_cls in clss
```